data-processing/node/README.md

```diff
@@ -1,62 +0,0 @@
-# Node scripts for data-processing
```

Comment:
Haha, yeah! One of the reasons we are refactoring this project is because it requires a complex combination of Python scripts, Node scripts, and Perl scripts, based on the languages that third-party utilities we want to use are written in. The vision is to have this project be mostly Python scripts, with Node parts and Perl parts separated out into standalone services.